### ISSUE TITLE
Need a minimal examples of Oauth and federation

### ISSUE THREAD

**Issue by LucienLee** [accounts-js/accounts#875]
Hi, I found this package from Graphql Module community. It looks neat and cool. 
However, the document is not well-prepared and neither for example when I have check issues and examples.  

Could anyone can give me some hints or minimal example of: 
- Is it OAuth strategy is ready to use? If it does, how to integrate 3rd party Oauth, like Google, with Graphql?  
- How to use it with the Apollo federation?
- What's the proper way to integrate with nextjs?

**Comment by agustif** [comment]
Hello @LucienLee there’s a PR by @nsine with a minimal working example of google-oauth. I made also a facebook one base off that. We’re still very much working in this, as it’s not quite ready just yet. But keep an eye for future releases as Im sure we will make it soin enough. If you are on a hurry and would like to see either example as is let me know and I can provide links to Pr’s / repo

**Comment by agustif** [comment]
Hello @LucienLee there’s a PR by @nsine with a minimal working example of google-oauth. I made also a facebook one based off that. We’re still very much working in this, as it’s not quite ready just yet. But keep an eye for future releases as Im sure we will make it soin enough. If you are on a hurry and would like to see either example as is let me know and I can provide links to the Pr’s / repo

**Comment by TeNNoX** [comment]
@agustif Yes, I'd love to see examples of OAuth use - I want to start using it and I'm ready for a bit of patchwork :sweat_smile:

**Comment by nsine** [comment]
hey @TeNNoX 
you can find some very draft version of example in this PR 🙂 https://github.com/accounts-js/accounts/pull/961

**Comment by TeNNoX** [comment]
I made it! :sunglasses: 

![Accounts-JS OAuth Demo](https://user-images.githubusercontent.com/2084639/84590525-591d2680-ae2f-11ea-8b32-4eaec9ece1f1.gif)

Would you fancy it if we would write the basic [docs for OAuth](https://www.accountsjs.com/docs/strategies/oauth)?

If so, what would be your requirements / tips?

I also thought about contributing my fullstack example - but it would be a bit more work I guess.

**Comment by pradel** [comment]
@TeNNoX any help is welcome for the OAuth docs, since they are non-existent for now, I think you can take a look at the graphql / password docs for inspiration :)

**Comment by TeNNoX** [comment]
We sadly had to pause our project, but now I'm going to write the basic docs.

FYI also @agustif
